oidc-server/src/main/resources/db/migration/V12___add_deadlock_indexes.sql

```diff
@@ -0,0 +1,23 @@
+ALTER TABLE `access_token` ADD INDEX `at_auth_holder_id` (`auth_holder_id`);
+ALTER TABLE `authentication_holder` ADD INDEX `user_auth_id_index` (`user_auth_id`);
+ALTER TABLE `authentication_holder_authority` ADD INDEX `aha_owner_id` (`owner_id`);
+ALTER TABLE `authentication_holder_extension` ADD INDEX `ahe_owner_id` (`owner_id`);
+ALTER TABLE `authentication_holder_resource_id` ADD INDEX `ahri_owner_id` (`owner_id`);
+ALTER TABLE `authentication_holder_response_type` ADD INDEX `ahrt_owner_id` (`owner_id`);
+ALTER TABLE `authentication_holder_scope` ADD INDEX `ahs_owner_id` (`owner_id`);
+ALTER TABLE `authentication_holder_request_parameter` ADD INDEX `ahrp_owner_id` (`owner_id`);
+ALTER TABLE `authorization_code` ADD INDEX `ac_auth_holder_id` (`auth_holder_id`);
+ALTER TABLE `saved_user_auth_authority` ADD INDEX `suaa_owner_id` (`owner_id`);
+ALTER TABLE `client_authority` ADD INDEX `ca_owner_id` (`owner_id`);
+ALTER TABLE `client_claims_redirect_uri` ADD INDEX `ccru_owner_id` (`owner_id`);
+ALTER TABLE `client_contact` ADD INDEX `cc_owner_id` (`owner_id`);
+ALTER TABLE `client_default_acr_value` ADD INDEX `cdav_owner_id` (`owner_id`);
+ALTER TABLE `client_grant_type` ADD INDEX `cgt_owner_id` (`owner_id`);
+ALTER TABLE `client_post_logout_redirect_uri` ADD INDEX `cplru_owner_id` (`owner_id`);
+ALTER TABLE `client_redirect_uri` ADD INDEX `creu_owner_id` (`owner_id`);
+ALTER TABLE `client_request_uri` ADD INDEX `cru_owner_id` (`owner_id`);
+ALTER TABLE `client_resource` ADD INDEX `cr_owner_id` (`owner_id`);
+ALTER TABLE `client_response_type` ADD INDEX `crt_owner_id` (`owner_id`);
+ALTER TABLE `client_scope` ADD INDEX `cs_owner_id` (`owner_id`);
+ALTER TABLE `refresh_token` ADD INDEX `rt_expiration` (`expiration`);
+ALTER TABLE `token_scope` ADD INDEX `ts_owner_id` (`owner_id`);
```

Comment:
The tables for the user_info (user_edu_*) have foreign keys that would benefit from an index as well (on user_id)

Reply:
Done. See https://github.com/OpenConext/OpenConext-oidc/blob/feature/db-indexes/oidc-server/src/main/resources/db/migration/V13___add_user_indexes.sql